tests/integration/cronenbroguelike/commands_test.py

```diff
@@ -0,0 +1,135 @@
+        mock_say.assert_called_once_with(
+            f"It is not possible to proceed {description}."
+        )
+
+    @patch("whimsylib.room.Room.on_exit")
+    @patch("whimsylib.globals.G.player.psyche.heal_or_harm")
+    def test_go_right_track(self, mock_heal, mock_exit, mock_say, mock_stdin):
+        ornate, squalid = _OrnateRoom(), _SqualidRoom()
+        ornate.add_character(G.player)
+        self.assertIs(ornate, G.player.current_room)  # Precondition
+
+        direction = directions.purple
+        description = direction.display_description
+        ornate.add_exit(direction, squalid)
+        next_room, _ = ornate.exit(description)
+        self.assertIs(squalid, next_room)  # Precondition
+
+        self._run_commands(mock_stdin, f"go {description}")
+
+        mock_exit.assert_called_once()
+
+        # Psyche is healed by 1 every time player enters a new room.
+        mock_heal.assert_called_with(1, do_log=False)
+
+        # This implies that squalid.on_enter() was called.
+        self.assertIs(squalid, G.player.current_room)
+        mock_say.assert_has_calls(
+            [call(f"You proceed {description}."), call(squalid.description)]
+        )
+
+    def test_look(self, mock_say, mock_stdin):
+        ornate, squalid = _OrnateRoom(), _SqualidRoom()
+        ornate.add_character(G.player)
+        self.assertIs(ornate, G.player.current_room)  # Precondition
+
+        ornate.add_item(Item("fruit bag, you know, of fruit"))
+
+        direction = directions.purple
+        description = direction.display_description
+        ornate.add_exit(direction, squalid)
+        next_room, _ = ornate.exit(description)
+        self.assertIs(squalid, next_room)  # Precondition
+
+        self._run_commands(mock_stdin, "look")
+
+        mock_say.assert_has_calls(
+            [
+                call("This room is very opulent, dang."),
+                call("There is a fruit bag, you know, of fruit lying on the ground."),
+                call("Exits are through a murky tunnel."),
+            ]
+        )
+
+    def test_stats(self, mock_say, mock_stdin):
+        self._run_commands(mock_stdin, "stats")
+        mock_say.assert_has_calls(
+            [
+                call("player", add_newline=True),
+                call("------", add_newline=False),
+                call("health    : 10/10", add_newline=False),
+                call("psyche    : 10/10", add_newline=False),
+                call("insanity  : 0/100", add_newline=False),
+                call("strength  : 10", add_newline=False),
+                call("stamina   : 10", add_newline=False),
+                call("------", add_newline=False),
+                call("- abilities -", add_newline=False),
+            ]
+        )
+
+    def test_different_stats(self, mock_say, mock_stdin):
+        G.player = actor.create_actor(20, 20, 20, 20, "the flippin devil")
+        G.player.health.heal_or_harm(-9)
+        G.player.add_ability(ability.meditation)
```

Comment:
Should this be:
```
         G.player.add_ability(ability.meditation())
```
?
No idea how this is passing right now.

Reply:
It worked for a very bad reason.

Reply:
I have made a comment on an open issue that will address this problem.